subprojects/groovy-json/src/main/java/groovy/json/JsonOutput.java

```diff
@@ -61,7 +61,8 @@
     private static final String NULL_VALUE = "null";
     private static final String JSON_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ";
     private static final String DEFAULT_TIMEZONE = "GMT";
-
+    private static final SimpleDateFormat dateFormat = new SimpleDateFormat(JSON_DATE_FORMAT);
```

Comment:
SimpleDateFormat is not thread-safe (see http://stackoverflow.com/questions/6840803/simpledateformat-thread-safety for an explanation)